irohad/multi_sig_transactions/mst_types.hpp

```diff
@@ -26,5 +26,15 @@ namespace iroha {
   using TransactionType = std::shared_ptr<iroha::model::Transaction>;
   using ConstPeer = const iroha::model::Peer;
   using TimeType = iroha::model::Transaction::TimeType;
+
+  class MstState;
+
+  template<typename T>
+  using ConstRefT = const T &;
+
+  using ConstRefTransaction = ConstRefT<TransactionType>;
+  using ConstRefPeer = ConstRefT<iroha::model::Peer>;
+  using ConstRefTime = ConstRefT<TimeType>;
+  using ConstRefState = ConstRefT<MstState>;
```

Comment:
This file looks quite odd here. These are more looks like just "types" rather than mst-ones

Reply:
What is your suggestion in this case?

Reply:
Idk, just comment out the situation.
imo it's better to leave this as is
and then add issue for moving this to _libs/common/types.hpp_ and refactor whole projects

Reply:
Those using useful only for MST. If you move it into a shared file, like types.hpp. All components will depend on MST. Not so good.

Reply:
All except `ConstRefState` barely connected with mst